src/less/components/_feedback-menu.less

```diff
@@ -0,0 +1,26 @@
+// Feedback menu
+//
+// Menu provides affixes to feedback and get help form 
+//
+// Markup:
+// <div class="feedback-menu">
+//  <span class="hyicon hyicon-feedback button-icon"></span>
+//  <span class="hyicon hyicon-gethelp button-icon"></span>
+// </div>
+//
+// Component 5.5
+
+.feedback-menu {
+  float: left;
+  font-size: 35px;
```

Comment:
menu should be stick to right-bottom edge of screen, so here should i add code responsible for that? 
```
position: relative;
z-index: 1000;
right: 0;
bottom: 100px;
```
or these values should be defined by user in own css?